docs/configuration/README.md

```diff
@@ -14,11 +14,12 @@
 
 **Optional Fields**
 
-- `jira_team_domain` - **Required** if you use the Jira issue tracker. Namespace of the Jira your workspace (e.g. `jira.myteamnamespace.com`, **myteamnamespace** would be the value for this key).
+- `enterprise_host` - **Required if** you use Enterprise Github. This is where you would supply the custom domain. (e.g. https://git.myCompany.com, `/api/v3` will be appended to this **do not include it for this value**).
```

Comment:
This `enterprise_host` was missing in the docs and you meant to add it here?

Reply:
If you look further down in the diff, I just moved it up.